kofta/public/locales/pt-BR/translation.json

```diff
@@ -84,10 +84,10 @@
       "followers": "seguidores",
       "following": "seguindo",
       "followHim": "seguir",
+      "unfollow": "deixar de seguir",
       "followingHim": "seguindo",
-      "copyProfileUrl": "copy profile url",
-      "urlCopied": "URL copied to clipboard",
-      "unfollow": "Unfollow"
```

Comment:
You removed this one but didn't readd it with the translation.

Reply:
unfollow key? check line 87. I changed the key order to match the same as english

Reply:
Oh I missed that. All good then. I'll merge this.